### ISSUE TITLE
wrong output!pls help me!

### ISSUE THREAD

**Issue by lohithareddym** [EvilPort2/Sign-Language#19]
Everything is working fine without any error byt whatever the gesture is given as input the outpput is always 5 ,what to do to get the correct output,even the accuracy of the model displayed during training is 0.0224  pls help me!

**Comment by kirtigodani** [comment]
you need to first run the load_images python file

**Comment by suhaas14** [comment]
getting some error in cnn_tf.py file.
type error in line 60 input

**Comment by lohithareddym** [comment]
Can u help me @kirtigodani

**Comment by lohithareddym** [comment]
Yes I'll get that error because of some version problems I tried to cast the type using tf.cast but no use try training the data using keras instead

**Comment by suhaas14** [comment]
can we do it in tensorflow only

**Comment by suhaas14** [comment]
object has no attribute shape

**Comment by EvilPort2** [comment]
I have checked the model. It is working fine for me. I will update the code nonetheless

**Comment by lohithareddym** [comment]
i havent changed any code and executed the files in the same order what might be the problem

**Comment by suhaas14** [comment]
kritigodani i need your help

**Comment by suhaas14** [comment]
pydot failed to call graphviz
i have installed both the error still occurs

**Comment by EvilPort2** [comment]
Please download the latest repo. I have fixed those issues
